ontology/proco.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: ontology_detail
+id: proco
+title: "PROCO: Ontology of Process Chemistry"
```

Comment:
Would suggest against redundant acronyms in the human-readable label. The abbreviation as `PROCO` is implied by the prefix

Reply:
Agreed, I changed that now.